helpers/api_helper/json_api_response_test.go

```diff
@@ -28,46 +28,35 @@ func TestNewErrorResponseWithValidErrorResponseObject(t *testing.T) {
 func TestNewErrorResponseWithMissingSomeFieldOnErrorResponseObject(t *testing.T) {
 	errorResponseObject := &api_helper.ErrorResponseObject{
 		Detail: "test-detail",
-		Status: 999,
 	}
 	errorResponse := errorResponseObject.NewErrorResponse()
 	expectedResult := gin.H{
 		"errors": []api_helper.ErrorResponseObject{{
 			Detail: "test-detail",
-			Status: 999,
+			Status: 0,
 		}},
 	}
 
 	assert.Equal(t, expectedResult, errorResponse)
 }
 
-func TestConstructDataResponseWithValidDataResponseObject(t *testing.T) {
-	dataResponseObject := &api_helper.DataResponseObject{
+func TestGetRelationshipsWithValidDataResponseObject(t *testing.T) {
+	dataResponseObject := api_helper.DataResponseObject{
 		ID:   "1",
 		Type: "test",
```

Comment:
As recommended above, all response attributes must be stable and always returned.

Reply:
Fixed in 7960a60637d858217f33899385e892a0abd5eae6